### PR TITLE
Changelog updated for 3.2.0 release

### PR DESCRIPTION
**Pull request by SaurabhMSFT**
Changelog updated for 3.2.0 release

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+3.2.0
+=========
+### Breaking change
+- IDownstreamApi overloads take a JsonTypeInfo<T> parameter to enable source generated JSON deserialization.[#3017]
```
Comment (jmprieur) [inline changelog.md:4]:
```suggestion
- .In .NET 8 and above, IDownstreamApi overloads take a JsonTypeInfo<T> parameter to enable source generated JSON deserialization.[#3017]
```

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+### Bug fixes:
+- Azure region is used while creating application keys for caching application objects [#3002]
```
Comment (jmprieur) [inline changelog.md:7]:
```suggestion
- Azure region is used while creating application keys when the TokenAcquisition service caches application objects, and the TokenAcquirerFactory caches TokenAcquirer  [#3002]
```

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @SaurabhMSFT

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+3.2.0
+=========
+### Breaking change
```
Comment (jennyf19) [inline changelog.md:3]:
```suggestion
- Updated to Microsoft.Identity.Abstractions 7.0.1
- Updated to Microsoft.IdentityModel.* 8.1.0
- Updated to Microsoft.Identity.Client 4.64.1
 
### New features
```

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,16 @@
+3.2.0
+=========
+- Updated to Microsoft.Identity.Abstractions 7.0.1
+- Updated to Microsoft.IdentityModel.* 8.1.0
+- Updated to Microsoft.Identity.Client 4.64.1
+ 
+### New features
+- .In .NET 8 and above, IDownstreamApi overloads take a JsonTypeInfo<T> parameter to enable source generated JSON deserialization.[#3017]
```
Comment (jennyf19) [inline changelog.md:8]:
```suggestion
- In .NET 8 and above, `IDownstreamApi` overloads take a `JsonTypeInfo<T>` parameter to enable source generated JSON deserialization. See [#3017] for details.
```

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,16 @@
...
+=========
+- Updated to Microsoft.Identity.Abstractions 7.0.1
+- Updated to Microsoft.IdentityModel.* 8.1.0
+- Updated to Microsoft.Identity.Client 4.64.1
+ 
+### New features
+- In .NET 8 and above, `IDownstreamApi` overloads take a `JsonTypeInfo<T>` parameter to enable source generated JSON deserialization. See [#3017] for details.
+
+### Bug fixes:
+- Azure region is used while creating application keys when the TokenAcquisition service caches application objects, and the TokenAcquirerFactory caches TokenAcquirer  [#3002]
```
Comment (jennyf19) [inline changelog.md:11]:
```suggestion
- Azure region is used while creating application keys when the TokenAcquisition service caches application objects, and the TokenAcquirerFactory caches TokenAcquirer. See  [#3002] for details.
```

File: changelog.md [hunk 6]
```diff
@@ -1,3 +1,16 @@
...
+- Updated to Microsoft.IdentityModel.* 8.1.0
+- Updated to Microsoft.Identity.Client 4.64.1
+ 
+### New features
+- In .NET 8 and above, `IDownstreamApi` overloads take a `JsonTypeInfo<T>` parameter to enable source generated JSON deserialization. See [#3017] for details.
+
+### Bug fixes:
+- Azure region is used while creating application keys when the TokenAcquisition service caches application objects, and the TokenAcquirerFactory caches TokenAcquirer. See  [#3002] for details.
+
+
```
Comment (jennyf19) [inline changelog.md:13]:
```suggestion
### Fundamentals:
- Improved test coverage for `GetCacheKey`. See PR [#3020](https://github.com/AzureAD/microsoft-identity-web/pull/3020) for details.
- Update to .NET 9-RC1. See issue [#3025](https://github.com/AzureAD/microsoft-identity-web/issues/3025) for details.
- Fix static analysis warnings. See PR [#3024](https://github.com/AzureAD/microsoft-identity-web/pull/3024) for details.
```

File: changelog.md [hunk 7]
```diff
@@ -1,3 +1,19 @@
...
+=========
+- Updated to Microsoft.Identity.Abstractions 7.0.1
+- Updated to Microsoft.IdentityModel.* 8.1.0
+- Updated to Microsoft.Identity.Client 4.64.1
+ 
+### New features
+- In .NET 8 and above, `IDownstreamApi` overloads take a `JsonTypeInfo<T>` parameter to enable source generated JSON deserialization. See [#3017] for details.
+
+### Bug fixes:
+- Azure region is used while creating application keys when the TokenAcquisition service caches application objects, and the TokenAcquirerFactory caches TokenAcquirer. See  [#3002] for details.
```
Comment (jennyf19) [inline changelog.md:11]:
```suggestion
- Azure region is used while creating application keys when the TokenAcquisition service caches application objects, and the TokenAcquirerFactory caches TokenAcquirer. See [#3002] for details.
```

File: changelog.md [hunk 8]
```diff
@@ -1,3 +1,19 @@
+3.2.0
+=========
+- Updated to Microsoft.Identity.Abstractions 7.0.1
+- Updated to Microsoft.IdentityModel.* 8.1.0
+- Updated to Microsoft.Identity.Client 4.64.1
+ 
+### New features
+- In .NET 8 and above, `IDownstreamApi` overloads take a `JsonTypeInfo<T>` parameter to enable source generated JSON deserialization. See [#3017] for details.
```
Comment (jennyf19) [inline changelog.md:8]:
```suggestion
- In .NET 8 and above, `IDownstreamApi` overloads take a `JsonTypeInfo<T>` parameter to enable source generated JSON deserialization. See issue [#2930] for details.
```

**Pull request merged**